examples/internal/proto/examplepb/echo_service.pb.gw.go

```diff
@@ -289,11 +289,6 @@ func local_request_EchoService_Echo_2(ctx context.Context, marshaler runtime.Mar
 		return nil, metadata, status.Errorf(codes.InvalidArgument, "missing parameter %s", "lang")
 	}
 
-	if protoReq.Code == nil {
-		protoReq.Code = &SimpleMessage_Lang{}
-	} else if _, ok := protoReq.Code.(*SimpleMessage_Lang); !ok {
```

Comment:
Do we not need this `nil` check anymore? It seems to me like this could panic now.

Reply:
Yep, should be fixed now.